eslint.config.js

```diff
@@ -72,87 +58,19 @@ const config = [
                     fixStyle: "separate-type-imports",
                 },
             ],
-            "@typescript-eslint/consistent-type-exports": [
-                "error",
-                {
-                    fixMixedExportsWithInlineTypeSpecifier: false,
-                },
-            ],
+            "@typescript-eslint/consistent-type-exports": "error",
```

Comment:
[`@typescript-eslint/consistent-type-exports` defaults `fixMixedExportsWithInlineTypeSpecifier` to `false`](https://typescript-eslint.io/rules/consistent-type-exports#options).